mutiny-core/src/lib.rs

```diff
@@ -28,23 +28,13 @@ mod localstorage;
 mod logging;
 mod lspclient;
 mod node;
-mod nodemanager;
+pub mod nodemanager;
 mod peermanager;
 mod proxy;
 mod socket;
 mod utils;
 mod wallet;
 
-use log::{debug, Level};
```

Comment:
are we not gonna move nodemanager into here?

Reply:
I dont think that core's node manager needs to be in `lib.rs`. I thought it was a good suggestion for `mutiny-wasm` since it's essentially a wrapper, but `core/nodemanager.rs` is more complex and could use it's own file.